tests/containers.bats

```diff
@@ -45,6 +47,7 @@ load helpers
   cid2=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json busybox)
   run_buildah --log-level=error containers --noheading
   expect_line_count 2
+  [[ "$output" != "NAME"* ]]
```

Comment:
This does not do what you think it does.

It's clear that I need to add a 'not in output' test. Until I do so, please use:
```bash
    if [[ $output =~ NAME ]]; then
        expect_output "[no instance of 'NAME']" "buildah containers --noheading"
    fi
```
Likewise with lines 62-63 below.

Reply:
Please address this. As I wrote, this does not work: it does not do what you think it does. What it does is pass if `$output` does not begin with `NAME`. This will always pass, _always_, even if for some reason buildah emits a heading, because the heading does not begin with `NAME` - it begins with `CONTAINER ID`. What you **want** to do is make sure that `NAME` does not appear anywhere in the output. Again, please look into this more closely - this is a fundamental misunderstanding that is repeated in a few other places in your PR.